niworkflows/interfaces/timeseries1D.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Operations to apply analytic tools designed for 4D NIfTI time series to
+2D TSV files.
+"""
+import pandas as pd
+import numpy as np
+import nibabel as nb
+from nipype.utils.filemanip import fname_presuffix
+from nipype.interfaces.base import (
+    traits, TraitedSpec, BaseInterfaceInputSpec, SimpleInterface, File)
+from .filter import _unfold_image
+
+
+class TSVToImageInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='TSV file')
+    t_rep = traits.Float(mandatory=True, desc='Repetition time')
+
+
+class TSVToImageOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='Image derived from input TSV')
+    header = traits.CList(traits.Str, desc='Header stripped from TSV')
+
+
+class TSVToImage(SimpleInterface):
+    """Convert a TSV into an image"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'e')](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096913)